Dockerfile

```diff
@@ -4,7 +4,7 @@ ARG DISTBALL
 
 ENV ZB_HOME=/usr/local/zeebe \
     ZEEBE_LOG_LEVEL=info \
-    ZEEBE_GATEWAY_HOST=0.0.0.0 \
+    ZEEBEBROKER_GATEWAY_NETWORK_HOST=0.0.0.0 \
```

Comment:
that's correct

(I didn't change anything here, I just answered the question. 
If you do want changes, now would be the time to say it, because this would also affect cloud team and operate if we change it later)

Reply:
I would opt for the nested version, e.g.

```json
{
  "zeebe": {
    "broker": {
  }
}
```

@menski what do you think?

Reply:
will do that

Reply:
Wait, I'd like to hear what @menski thinks as well

Reply:
Talked to him, he agrees

Reply:
:+1:

Reply:
done